src/tests/suite/daos_pool.c

```diff
@@ -48,6 +48,68 @@ pool_connect_nonexist(void **state)
 	assert_int_equal(rc, -DER_NONEXIST);
 }
 
+const char *my_server_group = "daos_server";
+
+/** Connect to pool, connect and try to evict with a bad param */
+static void
+pool_evict(void **state)
+{
+	test_arg_t	*arg = *state;
+	daos_handle_t	 poh;
+	daos_event_t	 ev;
+	daos_pool_info_t info = {0};
+	int		 rc;
+
+	if (!arg->hdl_share && arg->myrank != 0)
+		return;
+
+	if (arg->async) {
+		rc = daos_event_init(&ev, arg->eq, NULL);
+		assert_int_equal(rc, 0);
+	}
+
+	arg->group = my_server_group;
+
+	if (arg->myrank == 0) {
+		/** connect to pool */
+		print_message("rank 0 connecting to pool %ssynchronously ... ",
+			      arg->async ? "a" : "");
+		rc = daos_pool_connect(arg->pool.pool_uuid, arg->group,
+				       &arg->pool.svc, DAOS_PC_RW, &poh, &info,
+				       arg->async ? &ev : NULL /* ev */);
+		assert_int_equal(rc, 0);
+		WAIT_ON_ASYNC(arg, ev);
+		assert_memory_equal(info.pi_uuid, arg->pool.pool_uuid,
+				    sizeof(info.pi_uuid));
+		/** TODO: assert_int_equal(info.pi_ntargets, arg->...); */
+		assert_int_equal(info.pi_ndisabled, 0);
+		print_message("success\n");
+	}
+
+	if (arg->hdl_share)
+		handle_share(&poh, HANDLE_POOL, arg->myrank, poh, 1);
+
+	if (arg->myrank == 0) {
+		print_message("sysname: %s\n", arg->group);
+		//rc = daos_pool_evict(pool_uuid, sysname, svc, NULL /* ev */);
+   	//print_message("rc: %d\n", rc);
```

Comment:
(style)  code indent should use tabs where possible
(style)  please, no space before tabs
(style)  Block comments use * on subsequent lines
(style)  do not use C99 // comments